From c8e57541dafd5eaefb69c98413641cf0285f3e53 Mon Sep 17 00:00:00 2001
From: Andreas Schmidt <andreas.schmidt@wago.com>
Date: Mon, 26 Feb 2018 13:31:44 +0100
Subject: [PATCH 161/261] dts: adapt mv88e6321 settings

* change switch mdio address to 0 (single chip)
* set phy address 0 for phy on port0
* set phy address 1 for phy on port1

Signed-off-by: Andreas Schmidt <andreas.schmidt@wago.com>
---
 arch/arm/boot/dts/wago-pfc200v3-devconf-0182.dts | 23 +++++++++++------------
 1 file changed, 11 insertions(+), 12 deletions(-)

diff --git a/arch/arm/boot/dts/wago-pfc200v3-devconf-0182.dts b/arch/arm/boot/dts/wago-pfc200v3-devconf-0182.dts
index b19b2df..72557c6 100644
--- a/arch/arm/boot/dts/wago-pfc200v3-devconf-0182.dts
+++ b/arch/arm/boot/dts/wago-pfc200v3-devconf-0182.dts
@@ -114,23 +114,22 @@
 	pinctrl-1 = <&davinci_mdio_sleep_pins>;
 	status = "okay";
 
-        phy_internal_1: ethernet-phy@2 {
+        ext_phy0: ethernet-phy@0 {
                 compatible = "ethernet-phy-ieee802.3-c22";
-                reg = <2>;
+                reg = <0>;
         };
 
-        phy_internal_2: ethernet-phy@5 {
+        ext_phy1: ethernet-phy@1 {
                 compatible = "ethernet-phy-ieee802.3-c22";
-                reg = <5>;
+                reg = <1>;
         };
 
-
-        phy3: ethernet-phy@3 {
+        int_phy3: ethernet-phy@3 {
                 compatible = "ethernet-phy-ieee802.3-c22";
                 reg = <3>;
         };
 
-        phy4: ethernet-phy@4 {
+        int_phy4: ethernet-phy@4 {
                 compatible = "ethernet-phy-ieee802.3-c22";
                 reg = <4>;
         };
@@ -139,7 +138,7 @@
                 compatible = "marvell,mv88e6085";
                 #address-cells = <1>;
                 #size-cells = <0>;
-                reg = <6>;
+                reg = <0>;
                 dsa,member = <0 0>;
                 reset-gpios = <&io_expander_70 4 GPIO_ACTIVE_LOW>;
                 phy-scan;
@@ -151,14 +150,14 @@
                         port@0 {
                                 reg = <0>;
                                 label = "ethX11";
-                                phy-handle = <&phy_internal_1>;
+                                phy-handle = <&ext_phy0>;
                                 phy-mode = "sgmii";
                         };
 
                         port@1 {
                                 reg = <1>;
                                 label = "ethX12";
-                                phy-handle = <&phy_internal_2>;
+                                phy-handle = <&ext_phy1>;
                                 phy-mode = "sgmii";
                         };
 
@@ -175,14 +174,14 @@
                         port@3 {
                                 reg = <3>;
                                 label = "ethX1";
-                                phy-handle = <&phy3>;
+                                phy-handle = <&int_phy3>;
                                 phy-mode = "rmii";
                         };
 
                         port@4 {
                                 reg = <4>;
                                 label = "ethX2";
-                                phy-handle = <&phy4>;
+                                phy-handle = <&int_phy4>;
                                 phy-mode = "rmii";
                         };
 
-- 
2.7.4

